services/comparison.py

```diff
@@ -11,7 +11,7 @@
 import minio
 import pytz
 from asgiref.sync import async_to_sync
-from django.db.models import Prefetch, QuerySet
+from django.db.models import Prefetch, Q, QuerySet
```

Comment:
Oh I can probably remove that import.  I previously had the `Q(report_type=None) | Q(report_type="coverage")` condition in here but have since moved it to `reports/managers.py`